### ISSUE TITLE
Контроллеры: конфликт нейминга

### ISSUE THREAD

**Issue by north-leshiy** [bitrix24/framework3-prototype#17]
Под контроллерами мы обычно все же принимает некоторое другое значение.

Кажется если мы будем называть "классические контроллеры" и "контроллер компонента" - мы усложним себе жизнь.

**Comment by overlinkbx** [comment]
Согласен.
Это только для тех кто сейчас плотно сидит на битриксе, это более менее очевидно. Потому что там из компонента сделали контроллер. Но, если рассматривать задумку тещуего битрикса, то там это смотрится вполне разумно. Все привыкли, что контроллер - это главный управленец некого энтрипоинта, и для борьбы с бесконечными аякс файлами - задумка вполне хорошая. Но если рассматривать bf3, то тут суть контроллера вообще теряется. К примеру Solution который подключает 3 контроллера? Да тут Solution сам больше похож на контроллер, чем компонент.
Если это сделано ради однообразного нейминга и в угоду psr4, то лучше стоит назвать вещи своими именами -"Component" и пусть он как и раньше, наследуюется от контроллера для реализации поведения контроллера как отдельной единицы.

Так же, лично для меня, стоит глобальный вопрос "Что такое компонент?". Я боюсь, что компонент всё ещё остаётся перегруженной архитектурной еденицей. Он у нас и виджет, и сервис, и контроллер.

**Comment by dmbitrix** [comment]
Да, тут скорее вопрос нейминга. В нашем случае имелось в виду что-то вроде RequestComponentController.
